templates/include/standard-space-infra.jsonnet

```diff
@@ -32,19 +35,34 @@ local prometheus = import "prometheus.jsonnet";
       "name": "standard-space-infra"
     },
 
-    "objects": admin.services("${ADDRESS_SPACE}") + [
+    local storage_templates = [
+      storage.template(false, false),
+      storage.template(false, true),
+      storage.template(true, false),
+      storage.template(true, true)
+    ],
+
+    local storage_template_configmap = [
+      templateConfig.storage("enmasse-storage-templates")
+    ],
+
+    local template_config = (if use_template_configmap then "enmasse-storage-templates" else ""),
+
+    "objects": [
+      standardController.deployment(template_config, "authservice-ca"),
```

Comment:
Is there a reason you prefer this outside the admin pod?

Reply:
No, I can move it in there instead. It was a bit easier to experiment with it separately.

Reply:
I have no objection to merging